charts/gardener/resource-manager/charts/runtime/templates/configmap.yaml

```diff
@@ -87,13 +87,13 @@ data:
         {{- if .Values.global.config.controllers.health.syncPeriod }}
         syncPeriod: {{ .Values.global.config.controllers.health.syncPeriod }}
         {{- end }}
-      kubeletCSRApprover:
-        enabled: {{ .Values.global.config.controllers.kubeletCSRApprover.enabled }}
-        {{- if .Values.global.config.controllers.kubeletCSRApprover.concurrentSyncs }}
-        concurrentSyncs: {{ .Values.global.config.controllers.kubeletCSRApprover.concurrentSyncs }}
+      csrApprover:
```

Comment:
Does it make sense to add a breaking release note for the GRM? However, I am not sure if someone actually deploys it using this chart.